test/missing.jl

```diff
@@ -217,11 +218,26 @@ end
     @test !(Union{Int, Missing}[1] != Union{Float64, Missing}[1.0])
     @test !(Union{Int, Missing}[1] != [1.0])
     @test !(Union{Bool, Missing}[true] != BitArray([true]))
+    @test [missing, 1] != [missing, 2]
     @test Union{Int, Missing}[1] != [2]
     @test [1] != Union{Int, Missing}[2]
     @test Union{Int, Missing}[1] != Union{Int, Missing}[2]
 end
 
+@testset "== and != on tuples" begin
+    @test ismissing((1, missing) == (1, missing))
+    @test ismissing(("a", missing) == ("a", missing))
+    @test ismissing((missing,) == (missing,))
+    @test ismissing((missing, 2) == (1, missing))
+    @test !((missing, 1) == (missing, 2))
```

Comment:
I could be missing something super obvious, but how is this different than the test a few lines down that does `!=`?

Reply:
Yeah, it's the same since the fallback is defined using `==`, but that's the assumption we're checking. :-)